Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "/workspace/.pyenv_mirror/user/current/lib/python3.12/site-packages/django/http/request.py", line 150, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io'. You may need to add '8000-hannahro15-july24hackat-zaupaau2dpw.ws-eu115.gitpod.io' to ALLOWED_HOSTS.
